### PR TITLE
adding option to set default localtime offset to normalize logs with

### PR DESCRIPTION
**Pull request by sergey-stratoscale**
@Stratoscale/systegration

### CODE REVIEW

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -219,10 +222,18 @@ def printLogs(logFiles, formatter):
...
+        conf[field] = value
+    with open('/tmp/stratolog-conf.tmp', 'w') as f:
+        f.write(yaml.dump(conf))
+    os.system('sudo mv /tmp/stratolog-conf.tmp %s' % LOG_CONFIG_FILE_PATH)
+
 if __name__ == "__main__":
     import argparse
     parser = argparse.ArgumentParser()
-    parser.add_argument("logFiles", metavar='logFile', nargs='+', help='logfiles to read or - for stdin')
+    parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read or - for stdin')
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: why '*'?

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -237,8 +248,20 @@ def printLogs(logFiles, formatter):
     parser.add_argument("--withThreads", action="store_true", help='print process and thread name')
     parser.add_argument("-f", "--follow", action="store_true", help='follow file forever', default=False)
     parser.add_argument("-l", "--localtime", action="store_true", help='print logs in localtime (default utc)', default=False)
+    parser.add_argument("--setLocaltimeOffset", type=int, help='set custom localtime offset in hours')
+    parser.add_argument("--restoreLocaltimeOffset", action="store_true", help='restore localtime offset to machine\'s offset')
     args = parser.parse_args()
 
+    actionHappened = False
+
+    if args.setLocaltimeOffset != None:
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: not sure what the logic is

File: py/strato/common/log/log2text.py (hunk 1 above)
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: if you want to just update the config file, you won't have to pass positional arg

File: py/strato/common/log/log2text.py (hunk 2 above)
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: if the flag is set to something, it'll update the config file. pretty straight forward.

File: py/strato/common/log/log2text.py [hunk 3]
```diff
@@ -54,7 +55,10 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
         self._clock = self._relativeClock if relativeTime else self._absoluteClock
         self._relativeClockFormat = "%.6f" if microsecondPrecision else "%.3f"
         self._minimumLevel = logging.INFO if noDebug else logging.DEBUG
-        self._localTimezoneOffset = lineparse.getTimezoneOffset()
+        if self.configFile['defaultTimezone'] != None:
```
Comment (mickey-stratoscale) [inline py/strato/common/log/log2text.py]: if self.configFile['defaultTimezone']:

**Comment by mickey-stratoscale** [comment]
shipit

**Comment by romang-stratoscale** [comment]
ship it

**Pull request closed without merging**